images.go

```diff
@@ -442,6 +487,9 @@ func (r *imageStore) BigDataNames(id string) ([]string, error) {
 }
 
 func (r *imageStore) SetBigData(id, key string, data []byte) error {
+	if key == "" {
+		return errors.Wrapf(ErrInvalidBigDataName, "data name can not be empty")
```

Comment:
Should we put the name of the function into the error.
Can't SetBigData, data name can not be empty.